### PR TITLE
Checking linters html and css

### PR DESCRIPTION
**Pull request by adelphinsucasa**
Hello there,

In this project, I have set up a "Hello_Microverse" project repository. My tasks include:

Documented a professional README.md.
Created a new branch according to the GitHub flow rules.
Included a .gitignore file.
Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
Added a descriptive README file to the project.
Added 3 files (each one in a separate commit):
A .html file that includes a header with the text "Hello Microverse!".
A .css file that includes one class with styles for your header.
A test.md file with the text "This file should be ignored by git".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,145 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://github.com/adelphinsucasa/Hello_Microverse.git)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (mukhammadrizooff) [inline README.md]: - [x] [OPTIONAL] If you have not deployment link of your project it would be better remove the this part 😃

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @adelphinsucasa, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
- [x] [REQUIRED] Good job so far! Please add more information about this milestone to your `Pull Request` Description because `Pull Request` should have a descriptive `title` telling you what changes are inside and `summary` describing introduced changes in `more detail`🙏 

image

- [x] [REQUIRED] Good job so far! Remember, `commit` messages should be `meaningful` and captalize`, please implement my above requirements in the next commit messages🙏 

image

- [x] [REQUIRED] Good job so far! There should be a `.gitignore` file listing `test.md` file and `node_modules` directory but I can't see in your `commit history` you did this implementations. Please implement this practice 🙏 

image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @adelphinsucasa, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Good Highlights

- Linter checks are passing
- Good commit messages
- Requested changes have been implemented
- Proper use of GitHub flow


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (adelphinsucasa) [reply README.md]: Great, you are right. I have removed this section. Thank you

**Comment by adelphinsucasa** [comment]
Hi
All the comments are good and I learned about them.
Thank you

**Pull request merged**